### PR TITLE
speech.getIndentationSpeech: use BeepCommand

### PR DESCRIPTION
**Pull request by LeonarddeR**
### Link to issue number:
Follow up of #10371

### Summary of the issue:
In #10371, speech.getIndentationSpeech was changed to return a sequence instead of a string. However, when indentation reporting with tones was on, we still called tones.beep directly.

### Description of how this pull request fixes the issue:
Use BeepCommands instead. This is just a small change, mainly to prove that this code now works as expected.

### Testing performed:
Tested that indentation speech and tones still work. Tested that no beep is played when exceeding 78 spaces.spaces or more.

### Known issues with pull request:
I'm currently in doubt as of whether we should use a speech sequence when speech mode is set to beeps.

### Change log entry:
None needed

### CODE REVIEW

File: source/speech/__init__.py [hunk 1]
```diff
@@ -487,22 +487,22 @@ def splitTextIndentation(text):
 
 
 def getIndentationSpeech(indentation: str, formatConfig: Dict[str, bool]) -> SpeechSequence:
-	"""Retrieves the phrase to be spoken for a given string of indentation.
+	"""Retrieves the indentation speech sequence for a given string of indentation.
 	@param indentation: The string of indentation.
 	@param formatConfig: The configuration to use.
-	@return: The phrase to be spoken.
```
Comment (derekriemer) [inline source/speech/__init__.py]: Why was this deleted?

**Review by derekriemer — APPROVED** [review]

File: source/speech/__init__.py (hunk 1 above)
Comment (leonardder) [reply source/speech/__init__.py]: It's already pretty clear from the first sentence of the doc string what this returns. It also no longer returns a phrase, and saying that this returns a speech sequence isn't necessary because the function is type annotated.

**Review by michaelDCurran — APPROVED** [review]

**Comment by derekriemer** [comment]
the indent beep can occur after the role announcement. This is kind of unsettling to me.

1. paste into browser, read this text.

```
data:text/html,<div><pre><a href="test.ignore">good night ladies,</a><br/><a href="https://nonothing.ignore">    Good bye ladies!</a><br/><a href="test.fm">Test test ladies.</a></pre></div>
```

The beeps occur after the word link. This is weird compared to the old version, but it isn't wrong.

**Comment by derekriemer** [comment]
@jcsteh what do you think about the beeps being delayed until NVDA says "link"?

**Comment by jcsteh** [comment]
I replied to this query via email, but apparently it didn't get here:

It's certainly different and I did notice it. It's technically correct according to how NVDA reads things. If the textweren't indentation (for example, if we read spaces all the time), you'd hear "link  space". In short,I think it's fine, though it will definitely feel different to people initially.
